pkgs/development/python-modules/termplotlib/default.nix

```diff
@@ -19,13 +19,26 @@ buildPythonPackage rec {
   };
 
   format = "pyproject";
-  checkInputs = [ pytestCheckHook numpy exdown gnuplot ];
+  checkInputs = [
+    pytestCheckHook
+    exdown
+    gnuplot
+    numpy
```

Comment:
```suggestion
```
those should not be duplicated.

Reply:
Surprisingly, the tests fail with `No such file or directory: 'gnuplot'` if I remove those :thinking:

Reply:
The library needs `gnuplot` to work properly, but it doesn't seem like the binary is propagated correctly. Do you know how to fix that @SuperSandro2000?

```
$ nix-shell -I nixpkgs=./. -p 'python39Packages.termplotlib' --run 'gnuplot --version'
gnuplot 5.4 patchlevel 2
$ nix-shell -I nixpkgs=./. -p 'python39.withPackages (pkgs: [pkgs.termplotlib])' --run 'gnuplot --version'
/tmp/nix-shell-192163-0/rc: line 3: gnuplot: command not found
```